rcsb/utils/insilico3d/ModelReorganizer.py

```diff
@@ -211,11 +217,16 @@ def __rebuildEntryIds(self, dataContainer, internalModelId):
         Remove the following categories (or attributes) from the mmCIF completely:
         - _entry.ma_collection_id
         - _pdbx_database_status.*
-        - _database_2.*  (entire category loop)
+        - _database_2.*  (entire category loop)  - will repopulate afterwards
```

Comment:
@piehld `database_2` is removed here and repopulated afterwards. Did you retain this step because it is easier to manage it this way for both MA and AF models, considering AF has this category populated with incorrect vocabulary term and MA doesn't have it at all? Just want to clarify.